<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EquipmentWidget</class>
 <widget class="QWidget" name="EquipmentWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>EquipmentWidget</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QGroupBox" name="armorsBox">
     <property name="title">
      <string>Armors</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QCheckBox" name="armorsRandomStats">
        <property name="text">
         <string>Random Stats</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="armorsRandomAppearance">
        <property name="text">
         <string>Random Appearance</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QGroupBox" name="weaponsBox">
     <property name="title">
      <string>Weapons</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <widget class="QCheckBox" name="weaponsRandomStatsAndElement">
        <property name="text">
         <string>Random Stats And Element</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="weaponsRandomAppearance">
        <property name="text">
         <string>Random Appearance</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="0">
    <widget class="QGroupBox" name="othersBox">
     <property name="title">
      <string>Others</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QCheckBox" name="othersRandomStats">
        <property name="text">
         <string>Random Stats</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
